### ISSUE TITLE
Fullscreen osu! doesn't hide the taskbar

### ISSUE THREAD

**Issue by Skubers** [ppy/osu#24160]
### Type

Game behaviour

### Bug description

I tried changing the settings on both osu! and Windows, and it just doesn't want to hide it after getting focus
![image](https://github.com/ppy/osu/assets/82095491/705adec8-fc20-4486-b9b4-1c14ddb3d90e)

### Version

2023.621.0-lazer

### Logs

[performance.log](https://github.com/ppy/osu/files/11990632/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11990633/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11990634/updater.log)
[database.log](https://github.com/ppy/osu/files/11990635/database.log)
[input.log](https://github.com/ppy/osu/files/11990636/input.log)
[network.log](https://github.com/ppy/osu/files/11990637/network.log)

**Comment by Skubers** [comment]
nvm that was something with my pc

**Comment by speedyes** [comment]
I have the exact same issue on Windows 10 LTSC IoT build 19041. Only workaround I found for now is to change renderer to OpenGL, but that changes something in my screen configuration that causes a 4 second input scan from my monitor every alt-tab. Some time ago the game hid my taskbar just fine after several alt-tabs, but that doesn't seem to work now.

I've tried to export logs from the in-game setting, but it just errors out with red color.


<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/7a49320e-a2d1-47a9-a1b4-329a69fe26bc" />
